content/08_fractals.html

```diff
@@ -340,7 +340,7 @@ <h3 id="the-monster-curve">The “Monster” Curve</h3>
 <p>Where do I get these points? Why not ask the<code>KochLine</code> object to calculate them for me?</p>
 <pre class="codesplit" data-code-language="javascript">function generate() {
   let next = [];
-  for (let segment in lines) {
+  for (let segment of lines) {
```

Comment:
@shiffman this should be "segment of **segments**" I think

Reply:
How strange, I fixed it in Notion and I thought I started the build process afterwards, but maybe I'm confused. I'll merge this and run again.

<img width="822" alt="Screen Shot 2023-09-09 at 6 56 25 PM" src="https://github.com/nature-of-code/noc-book-2023/assets/191758/bb8dd171-372e-4cdd-b45a-3db56642bd75">

Reply:
Doesn't look fixed based on your Notion screenshot. There are no `lines` variable, it should be `segments`.

Reply:
Yes, sorry, my brain has clearly melted out of my ears, hah, I was confused and looking at the wrong thing!